src/Hazelcast.Net.Tests/Events/EventArgs.cs

```diff
@@ -14,6 +14,7 @@
 
 using System;
 using System.Collections.Generic;
+using Hazelcast.Clustering;
```

Comment:
```suggestion
```

Reply:
Don't worry too much about these. There will be another PR that essentially clears all these rogue using (it's done with an automatic tool).

Reply:
Ok, you're probably talking about ReSharper? :)